tests/integration/components/file-icon/component-test.ts

```diff
@@ -10,28 +10,125 @@ module('Integration | Component | file-icon', hooks => {
     test('default file icon', async function(assert) {
         this.set('item', EmberObject.create({}));
         await render(hbs`{{file-icon item=item}}`);
-        assert.dom('i').hasClass('fa-file-o');
+        assert.dom('svg').hasClass('fa-file');
     });
 
     test('file type not found uses default', async function(assert) {
         const file = EmberObject.create({ itemName: 'file.notafiletype' });
         this.set('item', file);
         await render(hbs`{{file-icon item=item}}`);
-        assert.dom('i').hasClass('fa-file-o');
+        assert.dom('svg').hasClass('fa-file');
     });
 
     test('file doesnt have a type, uses default', async function(assert) {
         const file = EmberObject.create({ itemName: 'doesntevenhaveatypelikewow' });
         this.set('item', file);
         await render(hbs`{{file-icon item=item}}`);
-        assert.dom('i').hasClass('fa-file-o');
+        assert.dom('svg').hasClass('fa-file');
     });
 
-    test('file gets the right icon for type', async function(assert) {
+    test('file gets the right icon for type - code', async function(assert) {
         const file = EmberObject.create({ itemName: 'normalfilefornormalpeople.c' });
         this.set('item', file);
         await render(hbs`{{file-icon item=item}}`);
-        assert.dom('i').hasClass('fa-file-code-o');
-        assert.dom('i').doesNotHaveClass('fa-file-o');
+        assert.dom('svg').hasClass('fa-file-code');
+        assert.dom('svg').doesNotHaveClass('fa-file');
+    });
+
```

Comment:
These tests were super-helpful, so I expanded them. We should prooobabbly do similar tests for other components that handle multiple icons.